#goal definition
gvgexploration/Ridge ridge
---
#result definition
string result
geometry_msgs/Pose[] explored_points
---
#feedback
string progress
geometry_msgs/Pose current_point